src/partials/layout.html

```diff
@@ -221,5 +221,54 @@
       });
     </script>
     <script async src="https://cdn.announcekit.app/widget-v2.js"></script>
+    <script>
+      filterSelection("all")
+      function filterSelection(c) {
+        var x, i;
+        x = document.getElementsByClassName("filterDiv");
+        if (c == "all") c = "";
+        // Add the "show" class (display:block) to the filtered elements, and remove the "show" class from the elements that are not selected
+        for (i = 0; i < x.length; i++) {
```

Comment:
When iterating over a list we can do `for (const card of integrationCards) {...}`.